packages/components/src/components/wizard/wizard.scss

```diff
@@ -0,0 +1,73 @@
+.wizard {
+    width: inherit;
+    height: inherit;
+
+    &__step {
+        width: 100%;
+        align-items: center;
+    }
+    &__header {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+
+        &-steps {
+            padding-bottom: 0.8rem;
+            width: 100%;
+            background: var(--general-section-2);
+
+            &-title {
+                color: var(--brand-red-coral);
+                font-size: 1.4rem;
+                font-weight: bold;
+                line-height: 20px;
+                padding: 1.6rem 2.4rem 0.8rem;
+            }
+            &-subtitle {
+                color: var(--text-prominent);
+                line-height: 18px;
+                font-size: 1.2rem;
+                font-weight: bold;
```

Comment:
(optional)(suggestion) there is a shortcode for this: 
`@include typeface(--small-left-bold-prominent)`